cli/src/pcluster/config/cluster_config.py

```diff
@@ -1503,6 +1519,7 @@ def __init__(
         self.managed_head_node_security_group = None
         self.managed_compute_security_group = None
         self.instance_types_data_version = ""
+        self._set_default_head_node_root_volume_size()
```

Comment:
What's the default value we are setting? Should this size be dependant on the additional software installed?

Reply:
nvm, clarified with the code below